### PR TITLE
Transition redirects first part

### PR DESCRIPTION
**Pull request by kathycarothers**
Redirects for transition for Press and RAD. I was only to get a couple in. I will have more to add

### CODE REVIEW

**Review by djgarr — APPROVED** [review]
The updates look good. Thanks!

File: redirects-transition.conf [hunk 1]
```diff
@@ -1336,6 +1342,8 @@ rewrite ^/pubrec/cfsdd/cfsdd.shtml https://www.fec.gov/introduction-campaign-fin
 rewrite ^/pubrec/cfsdd/cfsded_000.pdf https://www.fec.gov/resources/cms-content/documents/cfsded.pdf redirect;
 
 # rad/ redirects
+rewrite ^/rad/analyst_search.shtml https://www.fec.gov/help-candidates-and-committees/question-rad/ redirect;
+rewrite ^/rad/best_practices_to_avoid_pitfalls_2018.pdf https://www.fec.gov/help-candidates-and-committees/filing-reports/#troubleshooting redirect;
```
Comment (dorothyyeager) [inline redirects-transition.conf:1346]: Redirects to anchor links don't work. Need to take off the #troubleshooting part.

File: redirects-transition.conf [hunk 2]
```diff
@@ -1296,19 +1296,25 @@ rewrite ^/press/foia.shtml https://www.fec.gov/freedom-information-act/ redirect
 rewrite ^/press/index.shtml https://www.fec.gov/press/ redirect;
 rewrite ^/press/news_releases.shtml https://www.fec.gov/updates/?update_type=press-release redirect;
 rewrite ^/press/press_contacts.shtml https://www.fec.gov/press/#contact redirect;
+rewrite ^/press/PressOffice.xml https://www.fec.gov/updates/ redirect;
```
Comment (dorothyyeager) [inline redirects-transition.conf:1299]: Redirect to https://www.fec.gov/updates/?update_type=weekly-digest

**Review by dorothyyeager — CHANGES_REQUESTED** [review]
See comments.

File: redirects-transition.conf (hunk 2 above)
Comment (kathycarothers) [reply redirects-transition.conf:1299]: Going to keep as is

**Review by patphongs — APPROVED** [review]
👍

**Pull request merged**